sermant-plugins/sermant-mq-grayscale/mq-grayscale-rocketmq-plugin/src/main/java/io/sermant/mq/grayscale/config/RocketMqConsumerClientConfig.java

```diff
@@ -24,7 +24,7 @@
  * @author chengyouling
  * @since 2024-05-27
  **/
-public class MqConsumerClientConfig {
+public class RocketMqConsumerClientConfig {
```

Comment:
It is recommended that the configuration class and dynamic configuration class be extracted into a common module to facilitate gray consumption of other message queues in the future.